packages/config/src/Config.ts

```diff
@@ -1,12 +1,137 @@
 import JsonFile, { JSONObject } from '@expo/json-file';
-import fs from 'fs-extra';
 import path from 'path';
 import slug from 'slugify';
 
-import { AppJSONConfig, ExpRc, ExpoConfig, PackageJSONConfig, ProjectConfig } from './Config.types';
+import {
+  AppJSONConfig,
+  ExpRc,
+  ExpoConfig,
+  PackageJSONConfig,
+  Platform,
+  ProjectConfig,
+} from './Config.types';
 import { ConfigError } from './Errors';
+import { findAndEvalConfig } from './getConfig';
+import { getRootPackageJsonPath, projectHasModule } from './Modules';
 import { getExpoSDKVersion } from './Project';
 
+/**
+ * Get all platforms that a project is currently capable of running.
+ *
+ * @param projectRoot
+ * @param exp
+ */
+function getSupportedPlatforms(
+  projectRoot: string,
+  exp: Pick<ExpoConfig, 'nodeModulesPath'>
+): Platform[] {
+  const platforms: Platform[] = [];
+  if (projectHasModule('react-native', projectRoot, exp)) {
+    platforms.push('ios', 'android');
+  }
+  if (projectHasModule('react-native-web', projectRoot, exp)) {
+    platforms.push('web');
+  }
+  return platforms;
+}
+
+export function serialize(val: any): any {
```

Comment:
this should be part of `findAndEval` because if any property is a function we are executing it. it also makes `getConfig` a bit cleaner because it can just receive the fully evaluated config object when it falls `findAndEvalConfig` rather than having to operate over properties on it. this could be problematic if `findAndEvalConfig` ever gets used elsewhere and people forget to use `serialize`